docs/guides/0x-animate-your-visualizations.md

```diff
@@ -0,0 +1,391 @@
+    <section>
+        <button id="js-play-button">Play</button>
+        <button id="js-pause-button">Pause</button>
+        <input type="range" id="js-duration-range" min="0" max="30" step="1">
+    </section>
+</aside>
+```
+
+As seen above, there is an `id` attribute assigned to  each element giving us access to each one using JavaScript:
+
+```js
+const $progressRange = document.getElementById('js-progress-range');
+const $playButton = document.getElementById('js-play-btn');
+const $pauseButton = document.getElementById('js-pause-btn');
+const $durationRange = document.getElementById('js-duration-range');
+```
+
+> Note: this guide makes use of some conventions. When setting an `id` to an element that is going to be accessed via JavaScript, the `id` starts with `js`. In addition, when assigning the element (`const $progressBanner = document.getElementById('js-progress-banner')`), the JavaScript value starts with `$`, which indicates that it contains an HTML element.
+
+#### Add listening events 
+
+In this step, we will add listening events tied to the different  interaction buttons used  to control the animation. For example, we will tell the **Pause** button that it has to be ready and react when clicked  to pause the animation.
+
+```js
+$pauseButton.addEventListener('click', () => {
+    viz.variables.animation.pause();
+});
+```
+
+And the same applies to the **Play** button:
+
+```js
+$playButton.addEventListener('click', () => {
+    viz.variables.animation.play();
+});
+```
+
+In the same way, we can  update the **duration** of the animation when the duration range is adjusted in the slider:
+
+```js
+$durationRange.addEventListener('change', () => {
+    viz.variables.duration = parseInt($durationRange.value, 10);
+});
+```
+
+#### Update the progress
+
+You have learned how change the animation by interacting with the different buttons, and now you are going learn how to update the current progress of the animation. This function will be the one responsible of updating the progress range by updating the range value with the result of `getProgressPct()` method:
+
+```js
+function updateProgress () {
+    $progressRange.value = viz.variables.animation.getProgressPct();
+}
+```
+
+Let’s call this function periodically by using `setInterval`:
+
+```js
+setInterval(updateProgress, 100);
+```
+
+> Note: You can also do this by listening to layer events, you will learn how to use these events in the [Interactivity and Events Guide](09-interactivity-events.md)
+
+#### All together
+
+Well done! In the example below you will find the full code, including with the animation controls. You can play with it and make adjustments.
+
+Remember you can style your HTML controls using CSS. These styles are very simple, but at the beginning of this guide you have a complete example that includes beautiful styles. This example is also available in the [examples section](http://carto.com/developers/carto-vl/examples/#example-animation-controls).
```

Comment:
/developers url

Reply:
I see we don't have here the **/map/** fragment anymore... Have we get ridden of that? 

If the answer is YES, then we should talk a bit about when do we update all the affected urls in our guides